### DISCUSSION TITLE
how to run the project

### DISCUSSION THREAD

**Discussion by pandyaparth** [aws-samples/talk-dev-to-me-twitch#114] [Q&A] [answered]
Hi,
I'm new to the MFE and I'm trying to use this project/code to learn. I've clone the repo and I'm trying to run the micro-frontends-module-federation without any success. Can you please provide npm/yarn commands and sequence to start the project successfully on localhost:3001? I only see a blank page after I've done yarn and yarn start on MFE directories and then trying to start the appshell.
Thanks in advance.

**Comment by pandyaparth** [comment]
Here is the console screenshot after starting all the MFE and visiting localhost:3001.

<img width="1579" alt="image" src="https://user-images.githubusercontent.com/1484186/151745575-7b722526-e1c2-4ada-b54b-3caa3a78d6bc.png">

**Comment by walawe** [comment, marked as answer]
@pandyaparth I found the above issue are due to version of react-router-dom libary (v6) being used. I have created PR downgrading to v5.

**Reply by pandyaparth** [reply]
Thank you @walawe . I tested the changes from your pull request and it works as expected. :)
